examples/server/x11.c

```diff
@@ -11,7 +11,7 @@
 
 void dirty_copy(rfbScreenInfoPtr rfbScreen, const uint8_t* data, int width, int height, int nbytes);
 void convert_bgrx_to_rgb(const uint8_t* in, uint16_t width, uint16_t height, uint8_t* buff);
-void get_window_size(xcb_connection_t* conn, xcb_window_t window, uint16_t* width, uint16_t* height);
+void get_window_size(xcb_connection_t* conn, xcb_window_t window, int16_t* width, int16_t* height);
```

Comment:
Is this really needed? If not, please leave it out.

Reply:
Oops! Not needed...